### DISCUSSION TITLE
Turn docker "latest" tag to latest semver tag

### DISCUSSION THREAD

**Discussion by sp3nx0r** [renovatebot/renovate#15910] [Request Help] [answered]
We've got some scenarios where we're currently using `latest` tag and are looking for some wizardry to turn that latest into an actual version (note: not using digest hash). Would a clever regexManager be able to force the conversion of latest to semver? Didn't see this in existing Discussions docs. Cheers

**Comment by rarkins** [comment, marked as answer]
It's been a capability we've long though about, but not yet implemented. I'm not sure there's any regex manager trickery we could use, but you never know.

If we were to simply pick the latest stable semver tag, that would be pretty easy. If people wanted us instead to check that the digest SHA matches, it would be much harder, because we need one API call per tag to get each digest (unfortunately can't get them in bulk with any API we're aware of).

**Reply by JamieMagee** [reply]
One development that might make this more easily possible in the future: most major cloud platforms currently have custom APIs on top of the container registry standard. That includes searching tags by digest.

I don't think that we should build on top of custom APIs, but I wouldn't be surprised if we see that API make it's way into a future version of the container registry standard.
